src/main/docker/cypress/Coverage-Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM artemis:coverage-latest
```

Comment:
I'm not sure where this name and tag comes from. I think it should be 
```suggestion
FROM ghcr.io/ls1intum/artemis:latest
``` 

If your intention was to build and tag the image locally you should put that in the documentation :)

Reply:
Yes, the setup uses a locally built image. I will update the documentation 👍